docs/index.mdx

```diff
@@ -1,36 +1,42 @@
+<Card title="🎉 Widgetbook Cloud is now Live!" href="/cloud">
+  **Save 20% of your time** by shortening the feedback loop between designers
+  and developers.
+</Card>
+
 # Overview
 
-Discover Widgetbook, a powerful Flutter package inspired by Storybook.js that simplifies the process of cataloging widgets, testing them across various devices and themes, and sharing them effortlessly with designers and clients.
+Widgetbook is a Flutter package inspired by [Storybook.js](https://storybook.js.org/) that simplifies the process of cataloging widgets to create your own [design system](/glossary/design-system), and testing the widgets across various configurations, like different devices, languages and themes.
```

Comment:
I would remove the link to storybook.js. We don't want to lose people from our docs pages to non-widgetbook pages.